packages/teleport/src/features.ts

```diff
@@ -347,6 +348,32 @@ export class FeatureApps {
   }
 }
 
+export class FeatureKubes {
+  getTopNavTitle() {
+    return 'Kubernetes';
```

Comment:
`getTopNavTitle` is to display non-cluster specific titles (in place of the cluster selector), lets leave it as empty string